graddft_qnn/qnn_functional.py

```diff
@@ -69,6 +69,9 @@ def xc_energy(
         coefficients += mean
 
         coefficients = coefficients[:, jax.numpy.newaxis]  # shape (xxx, 1)
+        coefficients = jnp.concatenate(
```

Comment:
Should we learn 2 QNN, or just learn one and copy it twice. I don't know if the coeff of exchange and correlation must be the same?